modules/configuring-machine-pool-disk-volume-ocm.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa_nodes/rosa-managing-worker-nodes.adoc
+
+:_content-type: PROCEDURE
+ifdef::openshift-rosa[]
+[id="configuring_machine_pool_disk_volume_ocm{context}"]
+= Configuring machine pool disk volume using OpenShift Cluster Manager
+endif::openshift-rosa[]
+.Prerequisite for day 1
+* You have the option to select the sizing for the default machine pool during cluster installation.
+
+.Procedure for day 1
+
+. Navigate to {cluster-manager-url} to create your cluster.
+
+. Navigate to *Cluster Settings*.
+
+. Navigate to *Node pools*.
+
+. Under the *Advanced machine pool settings*, select the desired *Root disk size*.
```

Comment:
We don't have any "Advanced machine pool settings" in widget and remove the same
Select the desired Root disk size - should be sufficient
See the reference.
![image](https://github.com/openshift/openshift-docs/assets/93653094/0fe68ca5-0498-4418-9b0d-f9ecba830491)